simplq/src/components/common/Nav/LeftNav.jsx

```diff
@@ -16,13 +16,22 @@ const LeftNav = ({ open, toggleClose }) => {
   return (
     <div>
       <ul className={styles['left-nav']} open={open}>
+        {page === 'home' ? (
+          <li>
+            <a tabIndex={0} role="link" onKeyDown={scrollToHowItWorks} onClick={scrollToHowItWorks}>
+              How it works
+            </a>
+          </li>
+        ) : null}
```

Comment:
Displaying this only on home page for now, as the scroll doesn't work as it's not on home page.